### ISSUE TITLE
A question about BatchNormalization module

### ISSUE THREAD

**Issue by horserma** [torch/nn#952]
Hi,
I saw this
```
model = nn.SpatialBatchNormalization(m, nil, nil, false)
```
in the documentation. 
It seems we can use the BN without learning the scale and bias, but I am wondering if there is a way that I could learn the scale and bias using global statistics, which means I use the fixed moving mean and std but I want to learn the affine? Also, is it possible to view the means and stds of the BN module? Thanks

**Comment by fmassa** [comment]
@horserma One way to use a fixed mean and std is to train your network in `evaluate` mode. This way, the statistics won't be updated, but it is still possible to learn the affine transformation.
There was some discussion about that in https://github.com/torch/nn/issues/873

**Comment by horserma** [comment]
alright, thank you very much

**Comment by horserma** [comment]
@fmassa I forgot to ask. I am wondering about this "evaluate mode" and "training mode". Apart from the different behaviours of some layer modules, is there any difference? Normally, in evaluate mode, it requires less memory since it does not have to save all intermediate feature maps like in training mode. So in torch, is it similar?
